build.sbt

```diff
@@ -109,7 +109,8 @@ lazy val zioSchema = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies ++= Seq(
       "dev.zio" %% "zio"         % zioVersion,
       "dev.zio" %% "zio-streams" % zioVersion,
-      "dev.zio" %% "zio-prelude" % zioPreludeVersion
+      "dev.zio" %% "zio-prelude" % zioPreludeVersion,
+      "dev.zio" %% "zio-test"    % zioVersion
```

Comment:
I had to include this for the annotation definition, would it make sense for this annotation to live inside the zio-schema-zio-test project to avoid including this?

Reply:
Yes!